src/components/AddressField/AddressField.js

```diff
@@ -19,7 +19,7 @@ const AddressField = React.forwardRef(function AddressField(
           <div>
             <EthIdenticon
               address={address}
-              onCopy={onCopy}
+              onCopy={onCopy} // FIXME: Should be in YextCopy
```

Comment:
What do you mean by this?

Reply:
I typo, but the `onCopy` should be part of `TextCopy` component how you can see below.
![image](https://user-images.githubusercontent.com/691510/155768023-263adba2-6dcc-4093-8b66-b7b9bbb0d589.png)

The `EthIdenticon` dont have `onCopy` prop.

It is `encapsuled` with `TextCopy` and use with `adornoment` prop

![image](https://user-images.githubusercontent.com/691510/155768182-f7ef8d45-6004-4a45-8910-79aaccfb2bc9.png)

Reply:
Thank you for sharing more context. It is more clear now.